common.sh

```diff
@@ -273,17 +273,15 @@ export PULL_SECRET_FILE=${PULL_SECRET_FILE:-$WORKING_DIR/pull_secret.json}
 
 # Ensure a few variables are set, even if empty, to avoid undefined
 # variable errors in the next 2 checks.
-export PULL_SECRET=${PULL_SECRET:-}
 export PERSONAL_PULL_SECRET=${PERSONAL_PULL_SECRET:-$SCRIPTDIR/pull_secret.json}
 export CI_TOKEN=${CI_TOKEN:-}
 
-# avoid "-z $PULL_SECRET" to ensure the secret is not logged
-if [ ! -s ${PERSONAL_PULL_SECRET} -a ${#PULL_SECRET} = 0 ]; then
+if [ ! -s ${PERSONAL_PULL_SECRET} ]; then
   error "${PERSONAL_PULL_SECRET} is missing or empty"
   error "Get a valid pull secret (json string) from https://cloud.redhat.com/openshift/install/pull-secret"
```

Comment:
Currently if you run with an old config, you just hit this error, which doesn't mention PULL_SECRET is no longer supported, and it's not super-obvious if you've not been following these PRs - perhaps we could detect the case where PULL_SECRET is still set, and output a helpful message or link to the docs?

Reply:
Good point. Done.